src/argilla/labeling/text_classification/rule.py

```diff
@@ -143,6 +173,47 @@ def __call__(
             return self._label
 
 
+def add_rules(dataset: str, rules: List[Rule]):
+    """Adds the rules to a given dataset
+
+    Args:
+        dataset: Name of the dataset.
+
+    Returns:
+        api response
+    """
+    rules = [rule.convert_to_labeling_rule() for rule in rules]
+    return api.active_api().add_dataset_labeling_rules(dataset, rules)
+
+
+def delete_rules(dataset: str, rules: List[Rule]):
+    """Deletes the rules from the given dataset
+
+    Args:
+        dataset: Name of the dataset
+
+    Returns:
+        api response
+
+    """
+    rules = [rule.convert_to_labeling_rule() for rule in rules]
+    return api.active_api().delete_dataset_labeling_rules(dataset, rules)
+
+
+def update_rules(dataset: str, rules: List[Rule]):
+    """Updates the rules of the given dataset
+
+    Args:
+        dataset: Name of the dataset
+
+    Returns:
+        api response
+
+    """
+    rules = [rule.convert_to_labeling_rule() for rule in rules]
+    return api.active_api().update_dataset_labeling_rules(dataset, rules)
```

Comment:
the `api` methods return nothing. You can remove the `return` here

Reply:
fixed